test/evergreen.yml

```diff
@@ -154,67 +174,38 @@ functions:
           set -o verbose
           echo '#undef HAVE_FTRUNCATE' >> config/wiredtiger_config.h
     - *make_wiredtiger
-  "compile wiredtiger address sanitizer":
+  "run data validation stress test checkpoint":
+    - *fetch_artifacts
     - command: shell.exec
       params:
-        working_dir: "wiredtiger"
+        working_dir: "wiredtiger/cmake_build/test/checkpoint"
         shell: bash
         script: |
           set -o errexit
           set -o verbose
-          if [ "$OS" != "Windows_NT" ]; then
-            . test/evergreen/find_cmake.sh
-            if [ -d cmake_build ]; then rm -r cmake_build; fi
-            mkdir -p cmake_build
-            cd cmake_build
-            $CMAKE -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v4_clang.cmake -DCMAKE_C_FLAGS="-ggdb" -DWITH_PIC=1 \
```

Comment:
I didn't think `-DWITH_PIC=1` was a default, so we can't remove these (and the ones below)? I might be missing something here.

Reply:
It should have been a default from how we were actually using it from what I could tell. That was one of the changes I made in base.cmake.

Reply:
Ah you're right, I missed that small off -> on change. Thanks for clarifying.